### PR TITLE
Flb/fix to pylint2.15.4

### PR DESCRIPTION
**Pull request by francescalb**
# Description:
Pylint 2.15.4 introduced some new tests that were not followed. 
The code has been updated accordingly.

## Type of change:

- [x] Bug fix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist:

This checklist can be used as a help for the reviewer.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

### CODE REVIEW

File: ontopy/ontodoc.py [hunk 1]
```diff
@@ -392,14 +392,28 @@ def itemdoc(
...
+            for individual in item.instances():
+                print("item", item, "is a", item.is_a)
+                print(
+                    "individual",
+                    individual,
+                    "is instance of",
+                    individual.is_instance_of,
+                )
+
+                # print(item.individual.is_instance_of)
```
Comment (jesper-friis) [inline ontopy/ontodoc.py:404]: This could be removed

File: ontopy/ontodoc.py [hunk 2]
```diff
@@ -392,14 +392,28 @@ def itemdoc(
...
+
             for entity in [
                 _ for _ in item.instances() if item in _.is_instance_of
             ]:
+                print("entity", entity)
+                print("a", points)
```
Comment (jesper-friis) [inline ontopy/ontodoc.py:410]: Remove

File: ontopy/ontodoc.py [hunk 3]
```diff
@@ -392,14 +392,28 @@ def itemdoc(
...
                 points.append(
                     point_style.format(
                         point=asstring(entity, link_style), ontology=onto
                     )
                 )
+                print("b", points)
```
Comment (jesper-friis) [inline ontopy/ontodoc.py:416]: More debugging to remove

**Review by jesper-friis — CHANGES_REQUESTED** [review]
Looks good. You should only remove some leftover debugging statements

**Review by CasperWA — APPROVED** [review]

**Review by CasperWA — COMMENTED** [review]
Did you want to update the pylint requirements as well, by the way?

**Review by jesper-friis — APPROVED** [review]

**Pull request merged**